rotkehlchen/exchanges/coinbase.py

```diff
@@ -965,9 +1019,6 @@ def _deserialize_asset_movement(self, raw_data: dict[str, Any]) -> list[AssetMov
             if 'from' in raw_data:
                 event_type = HistoryEventType.DEPOSIT
 
-            if tx_type == 'send' and not amount_data['amount'].startswith('-'):
```

Comment:
where did this go?

Reply:
It is here now https://github.com/rotki/rotki/blob/be8eb56b47890b03c3ec8576b98122756a531087/rotkehlchen/exchanges/coinbase.py#L979